Project-18-Food-order-app/src/components/SuccessMessage/index.jsx

```diff
@@ -0,0 +1,34 @@
+import { useContext } from "react";
+import PropTypes from "prop-types";
+
+import Button from "../UI/Button";
+import { UserProgressContext } from "../../services/stores/UserProgress";
+
+function SuccessMessage({ onClose }) {
+  const { hideSuccessMessageDialog } = useContext(UserProgressContext);
+
+  const handleCloseButtonClick = () => {
+    hideSuccessMessageDialog();
+    onClose();
+  };
+
+  return (
+    <div>
+      <h2 className="modal-title">Success!</h2>
+      <p className="modal-desc">Your order was submitted successfully.</p>
+      <p className="modal-desc">
+        We will get back to you with more details via email within the next few
+        minutes.
+      </p>
+      <div className="modal-actions">
+        <Button onClick={handleCloseButtonClick}>Okay</Button>
+      </div>
+    </div>
+  );
+}
+
+SuccessMessage.propTypes = {
+  onClose: PropTypes.func,
```

Comment:
@sknepali You have used PropTypes to type-check your props, which is good. However, you can make it more specific. For instance, instead of PropTypes.func for onClose, you can use PropTypes.func.isRequired to ensure that this function is always passed.